lms/templates/courseware/accordion.html

```diff
@@ -6,73 +6,44 @@
 %>
 
 <%def name="make_chapter(chapter)">
-  <div class="chapter">
-      <%
-          if chapter.get('active'):
-              aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
-              active_class = ' class="active"'
-          else:
-              aria_label = chapter['display_name']
-              active_class = ''
-      %>
-      <h3 ${active_class} aria-label="${aria_label}">
-        <a href="#">
-          ${chapter['display_name']}
-        </a>
-      </h3>
-
-    <ul>
-      % for section in chapter['sections']:
-          <li class="${'active' if 'active' in section and section['active'] else ''} ${'graded'  if 'graded' in section and section['graded'] else ''}">
+<%
+if chapter.get('active'):
+    aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
+    active_class = 'active"'
+else:
+    aria_label = chapter['display_name']
+    active_class = ''
+%>
+<button class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-pressed="false">
+    <h3 class="group-heading ${active_class}" aria-label="${aria_label}">
```

Comment:
@clrux I think use of a Heading here is inappropriate.  Heading elements should be used to indicate the hierarchy of page content, not necessarily nav content.  I think this will result in too many Headings on the page, which reduces the effectiveness of navigating by Heading.  SR users who navigate by Heading expect to find content related to that heading following it, but here, they just find a list of links to subcontent. It makes sense to make these an ordered list